api/src/pcapi/routes/backoffice/venues/blueprint.py

```diff
@@ -418,6 +419,80 @@ def get_stats(venue_id: int) -> utils.BackofficeResponse:
     )
 
 
+def _get_venue_revenues_per_year(
+    venue_id: int,
+) -> dict[str, dict[str, float]]:  # {year: {"ind": amount, "ind": amount}
```

Comment:
```suggestion
) -> dict[str, dict[str, float]]:  # {year: {"ind": amount, "coll": amount}
```
Mais en fait je supprimerai le commentaire, ça se comprend assez vite dans le code